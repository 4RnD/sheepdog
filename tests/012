#!/bin/bash

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in 0 1 2; do
    _start_sheep $i
done

# add gateway node
_start_sheep 4 "-g"

sleep 1

echo check the number of vnodes
$COLLIE node list
