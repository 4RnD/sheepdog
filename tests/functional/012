#!/bin/bash

# Test gateway only functionality

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in 0 1 2; do
    _start_sheep $i
done

_wait_for_sheep 3

# add gateway node
_start_sheep 4 "-g"

_wait_for_sheep 4

echo check the number of vnodes
$COLLIE node list
