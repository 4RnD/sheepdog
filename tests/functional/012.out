QA output created by 012
check the number of vnodes
  Id   Host:Port         V-Nodes       Zone
   0   127.0.0.1:7000      	128          0
   1   127.0.0.1:7001      	128          1
   2   127.0.0.1:7002      	128          2
   3   127.0.0.1:7004      	 0          4
